pkg/jx/cmd/common/interfaces.go

```diff
@@ -0,0 +1,11 @@
+package common
+
+import (
+	"github.com/banzaicloud/bank-vaults/operator/pkg/client/clientset/versioned"
+	"k8s.io/client-go/kubernetes"
+)
+
+type NewCommonOptionsInterface interface {
+	KubeClient() (kubernetes.Interface, string, error)
+	VaultOperatorClient() (versioned.Interface, error)
```

Comment:
I would move this in the factory.go which is used as a facade for cmd.

Reply:
Unfortunately, `factory.go` is still inside the `cmd` package. Given that parts of the `cmd` package need to import the `vault` package, I needed to put this interface _outside_ the `cmd` package to prevent a circular dependency.